packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -108,11 +106,50 @@ export interface ITreeConfigurationOptions {
 	 * - More than one MapNode type: it's impossible to tell which map type is intended in the case of an empty map (`{}`).
 	 * - Both a MapNode and an ArrayNode: this case is not a problem for the canonical JSON representation, but is an issues when constructing from an Iterable, which is supported for both MapNode and ArrayNode.
 	 * - Both a MapNode and an ObjectNode: when the input is valid for the ObjectNode, the current parser always considers it ambiguous with being a MapNode.
-	 * - ObjectNodes which don't each have a unique required field: currently every ObjectNode type in a union must have a required field with a name that only it uses for any field.
+	 * - ObjectNodes which which have fields (require or optional) which include all required fields of another ObjectNode: currently every ObjectNodes are differentiated by the presence of their required fields.
 	 *
 	 * This check is conservative: some complex cases may error if the current simple algorithm can not show no ambiguity is possible.
 	 * This check may become more permissive over time.
 	 *
+	 * @example
+	 * This example shows an ambiguous schema, and how to disambiguate it using {@link Unhydrated} nodes:
+	 * ```typescript
+	 * const schemaFactory = new SchemaFactory("com.example");
+	 * class Feet extends schemaFactory.object("Feet", { length: schemaFactory.number }) {}
+	 * class Meters extends schemaFactory.object("Meters", { length: schemaFactory.number }) {}
+	 * const config = new TreeViewConfiguration({
+	 * 	// This combination of schema is can lead to ambiguous cases, and would error if preventAmbiguity is true.
+	 * 	schema: [Feet, Meters],
+	 * 	preventAmbiguity: false,
```

Comment:
```suggestion
	 * 	preventAmbiguity: true,
```

Or else the commented-out code below _won't_ fail, right?

Reply:
No. Regardless if this setting, its impossible to tell if `{ length: 5 }` is a Meters node or a Feet node, and toMap node will fail to parse it because of that.

If preventAmbiguity is set to true, the code below would never run, because constructing the configuration would throw an error: that's all preventAmbiguity does, it throws an error if the schema has ambiguous cases like that. It has no effect on how data is parsed (infect its impossible for it to impact parsing of data since the construction of Unhydrated nodes does parsing, and it doesn't know which tree/configuration applies to it).

This example exists to show what issues you can hit with ambiguous schema, and how to deal with them, so it must have preventAmbiguity: false otherwise we will be prevented from having the ambitious schema by an exception when making the configuration.

Reply:
Ah, okay. I find that confusing since this doc comment is about the `preventAmbiguity` flag, and the context of the comment just above this example is talking about the `preventAmbiguity` flag, but then we jump into this example where the `preventAmbiguity` flag is irrelevant. When this example says things like "this is invalid", it's not clear if it means "generally invalid" or "invalid when `preventAmbiguity` is enabled".

Reply:
I've clarified this in the example title.